CodeEdit/Features/NavigatorArea/SourceControlNavigator/SourceControlToolbarBottom.swift

```diff
@@ -23,8 +25,11 @@ struct SourceControlToolbarBottom: View {
 
     private var sourceControlMenu: some View {
         Menu {
-            Button("Discard Changes...") {}
-                .disabled(true) // TODO: Implementation Needed
+            Button("Discard Changes...") {
```

Comment:
Should we maybe say “Discard All Changes…”?

Reply:
SGTM!